crates/fsrt/src/test.rs

```diff
@@ -0,0 +1,30 @@
+mod tests {
+    use crate::{scan_directory_test, MockForgeProject};
+    use forge_loader::manifest::{ForgeManifest, FunctionMod};
+    use std::collections::HashMap;
+
+    #[test]
+    fn test_simple() {
+        let forge_manifest = ForgeManifest::create_manifest_with_func_mod(FunctionMod {
```

Comment:
I think you forgot to add this function.

Reply:
fixed! thank you :)